tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -622,7 +621,7 @@ describe(__filename, () => {
   it('does not render an admin link if no user is found', () => {
     const { store } = signInUserWithProps({
       userId: 1,
-      permissions: [USERS_EDIT, ADMIN_TOOLS],
+      permissions: [USERS_EDIT],
```

Comment:
So `Users:Edit` is enough now?

Reply:
Yes, `Admin:Tools` was removed in https://github.com/mozilla/addons-server/pull/15711.